### ISSUE TITLE
Add support for SSH and Wireguard public keys in P2K16

### ISSUE THREAD

**Issue by eliasbakken** [bitraf/infrastructure#148]
To facilitate automatic setup of SSH and Wireguard for new members.

**Comment by omega** [comment]
Support for adding/removing them to the user profile we are talking about here?

**Comment by eliasbakken** [comment]
Yes, support for the user to store his public keys in the database somehow,
and then we can fix adding the keys to heim etc in a separate step.
